themes/default/languages/english/Profile.english.php

```diff
@@ -419,7 +419,7 @@
 $txt['profile_warning_effect_watch'] = 'User will be added to moderator watch list.';
 $txt['profile_warning_effect_own_watched'] = 'You are on the moderator watch list.';
 $txt['profile_warning_is_watch'] = 'being watched';
-$txt['profile_warning_effect_moderation'] = 'All users posts will be moderated.';
+$txt['profile_warning_effect_moderate'] = 'All users posts will be moderated.';
```

Comment:
Should be either `user` or `user\'s`. But if I catch the meaning correctly I'd better be rewritten to something like `All posts by this user will be moderated.`